iroh-gateway/src/config.rs

```diff
@@ -27,7 +27,9 @@ pub struct Config {
     /// default port to listen on
     pub port: u16,
     /// flag to toggle whether the gateway should use denylist on requests
-    pub denylist: bool,
+    pub use_denylist: bool,
```

Comment:
heads up @Arqu, I've added a `use_` prefix to avoid using up the `denylist` configuration key on a boolean flag.